client/src/components/post/PostFooter/PostFooter.js

```diff
@@ -0,0 +1,32 @@
+
+import PostHashtag from "../PostHashtag/PostHashtag.vue";
+import AvatarSection from "../AvatarSection/AvatarSection.vue";
+export default {
+  name: 'post-footer',
+  components: {
+    PostHashtag,
+    AvatarSection
+  },
+  props: {
+    hashtags: Array,
```

Comment:
Instead of 

```
hashTags: Array
```

use
```
hashtags: []
```


While using "Array" name there could be clashes with some libraries